ui/src/app/utils/index.tsx

```diff
@@ -400,7 +400,7 @@ export const withRouteConfigData = () => {
 // of the CDR versions. This should only affect initial page loads, this HOC can
 // be included last (if multiple HOCs are in use) to minimize this impact.
```

Comment:
While this comment was true with `connectReplaySubject`, I don't think it's true anymore for `withStore` or the new subscribable, which appears to immediately render regardless of whether or not there's a value. Also, with the new store, there's a default "empty" value seeded into the store.

I suspect this causes a race condition on initial page render which can cause the initial page load to fail, e.g.:
https://app.circleci.com/pipelines/github/all-of-us/workbench/8720/workflows/2f20c5b4-98f7-4996-897d-3b3f7bda6a09/jobs/128585/artifacts

Possible resolutions:
- Make a different HOC which blocks child rendering until a store value is present, and remove the initial stub response from the store
- Refactor all of the clients of `withCdrVersions` to instead handle null/empty CDR versions (again, I think I would remove the stub response and use something like `null` or `undefined` to indicate that it's still loading).
- Move seeding of the cdr versions store somewhere early in the bootstrapping process, and block the main rendering path on it, e.g. in the signed in component or app router.